include/oneapi/dnnl/dnnl_graph.hpp

```diff
@@ -647,6 +655,21 @@ class tensor : public tensor_handle {
     }
 };
 
+/// Creates a tensor object for host-side scalar value. The data type contained
+/// in the logical tensor parameter will be used to interpret the scalar
+/// pointer. The property type in the logical tensor must be `host_scalar`.
+///
+/// @param lt The logical tensor describing the host scalar
+/// @param scalar The pointer to scalar value
+/// @returns Created tensor object
+inline tensor make_scalar_tensor(const logical_tensor &lt, void *scalar) {
+    dnnl_graph_tensor_t t = nullptr;
+    error::wrap_c_api(dnnl_graph_tensor_create(&t, &(lt.data), nullptr, scalar),
```

Comment:
(up to discussion) Would it be more error-proof to have a new C symbol as well?
For example, accepting nullptr for engine in dnnl_graph_tensor_create might make debugging harder on user side (they would get an error about host-side scalar or segfault if they pass nullptr as engine by mistake).